### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by hoim5833** [Nro001/Nro-Galaxy-Editor#5] [General]
![제목 없음](https://user-images.githubusercontent.com/128282717/227924309-438ebf94-1391-4a48-a7f4-ee164d56cf3c.png)

Modify the starting position, and when you play, there is a cloud like this and you can't erase it

**Comment by Scherzingetorix** [comment]
Hello,

I have a same issue, Besides the editor randomly freezing (a number of buttons are non-responsive, but I can live with that), I have a bigger problem. I liked the idea of pulling the starting system into the center of the galaxy, but unfortunately, it doesn't bring the borders with it, and as you can see in the picture, the stars remain gray; it doesn't discover the systems but leaves them in the Fog of War. Any ideas? BTW, I did everything according to the instructions. Addendum: if I run the survey console command, the problem is solved, but it ruins the game :D. Please help, how can this be fixed? Also, the discovered area remains at the original location of the starting system, so it stores it somewhere, it would just be good to know where.

![image](https://github.com/Nro001/Nro-Galaxy-Editor/assets/158042204/5fcfd650-568b-44de-bea5-5302d4b2865f)

**Comment by Scherzingetorix** [comment]
After several hours of struggling, I figured out what the problem is. There is a key called Terra_incognita that contains a data row, and this holds the data related to the undiscovered parts of the galaxy. It's a mystery exactly how to interpret the data stored in it. The workaround is to discover the galaxy using the survey command, then save this in a separate save file and extract the terra_incognita data row (first one belons to human player) from it, then insert it to edited gamestate file. I'm not sure yet how this affects the gameplay, but it solves the gray zone problem.
